src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/EligibilityAndSuitabilityServiceTest.kt

```diff
@@ -73,11 +73,13 @@ class EligibilityAndSuitabilityServiceTest {
 
       whenever(assessmentService.getCurrentAssessment(PRISON_NUMBER)).thenReturn(
         anAssessmentWithEligibilityProgress.copy(
-          suitabilityProgress = anAssessmentWithEligibilityProgress.suitabilityProgress.mapIndexed { i, it ->
-            if (i == 0) {
-              it.copy(status = SUITABLE, questions = listOf(it.questions[0].copy(answer = true)))
-            } else {
-              it
+          suitabilityProgress = {
+            anAssessmentWithEligibilityProgress.suitabilityProgress().mapIndexed { i, it ->
+              if (i == 0) {
```

Comment:
I hate these BTW.. mutability or presumably [lenses](https://arrow-kt.io/learn/immutable-data/lens/) would make these a bit easier to read